api/server/server.go

```diff
@@ -154,7 +154,7 @@ func NewServer(conf Config) (*Server, error) {
 	}
 
 	log.Info("Wiring internal components...")
-	mm, err := maxmind.New(filepath.Join(conf.MaxMindDBFolder, "./GeoLite2-City.mmdb"))
+	mm, err := maxmind.New(filepath.Join(conf.MaxMindDBFolder, "GeoLite2-City.mmdb"))
```

Comment:
This `conf.MaxMindDBFolder` is set either directly on a config struct or possibly from the new `powd` flag.